src/ServerDaemon.chpl

```diff
@@ -401,7 +401,14 @@ module ServerDaemon {
     
             this.connectUrl = this.getConnectUrl(this.serverToken);
             this.createServerConnectionInfo();
-            this.printServerSplashMessage(this.serverToken,this.arkDirectory);
+            if serverInfoNoSplash {
+                writeln("Server Information:");
+                writeln(getConfig());
```

Comment:
I'm not sure if we want to write `"Server Information:"` first or just write `getConfig()` since this mode should be primarily used by users who want easy access to the config info, maybe it would be easier to parse? I don't know much about this stuff though, so i'll defer to other reviewers expertise

Reply:
You make a good point, this will rarely be used out of context I wouldn't think, so I'll make this change for ease of parsing

Reply:
Updated